tensorboard/webapp/runs/views/legacy_runs_selector/legacy_runs_selector_component.ts

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {
+  AfterViewInit,
+  Component,
+  ElementRef,
+  EventEmitter,
+  ViewChild,
+  Output,
+} from '@angular/core';
+
+interface PolymerChangeEvent extends CustomEvent {
+  detail: {value: any};
+}
+
+@Component({
+  selector: 'tb-legacy-runs-selector-component',
+  template: `
+    <tf-runs-selector #selector></tf-runs-selector>
+  `,
+})
+export class LegacyRunsSelectorComponent implements AfterViewInit {
+  @Output()
+  onSelectionChange = new EventEmitter<string[]>();
+
+  @ViewChild('selector', {static: true})
+  private selector!: ElementRef;
+
+  ngAfterViewInit() {
+    this.selector.nativeElement.addEventListener(
+      'selected-runs-changed',
```

Comment:
I don't see any code that dispatches this event.  Is the event firing coming in a followup PR?  If not, could you please point to it in case I missed?

Reply:
Left a comment on where it is coming from. Hint: it is a Polymer thing :)